src/app/shared/components/next-days/next-days.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { WeatherService } from '../../services/weather.service';
+import { INextDays } from 'src/app/utilities/interfaces/nextdays.interface';
+import * as moment from 'moment';
+
+@Component({
+  selector: 'app-next-days',
+  templateUrl: './next-days.component.html',
+  styleUrls: ['./next-days.component.scss'],
+})
+export class NextDaysComponent implements OnInit {
+  @Input() forecastData: INextDays;
+  public day: any;
```

Comment:
any? create interfaces for this

Reply:
can't change because i recieve different type of data